### DISCUSSION TITLE
Serialization Generation for Game Development

### DISCUSSION THREAD

**Discussion by TrevorCash** [JhnW/devana#7] [Show and tell]
I started using devana for generating serialization functions for a game using the cereal c++ serialization library.  The game has alot of structs with alot of members.  Im using it to generate all the serialization functions using the struct definitions.   So far super useful and easy to work with.

**Comment by JhnW** [comment]
Hi.
Very nice to hear it. If you have any problems or questions, don't hesitate.

I am currently working on something like a code generation framework to be able to describe the C++ input code, but it will take some time before it reaches full production quality.
